### DISCUSSION TITLE
一直在Configuring x86-windows-rel

### DISCUSSION THREAD

**Discussion by XiaoYou0426** [microsoft/vcpkg#32710] [Q&A]
为什么会一直在这个步骤，几个小时，没有任何反应
Building gettext:x86-windows...
-- Installing port from location: C:\Users\15653\AppData\Local\vcpkg\registries\git-trees\4c583e9adc3ab9a044b241466cebedf162cdc410
-- Downloading https://ftp.gnu.org/pub/gnu/gettext/gettext-0.21.1.tar.gz;https://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gettext/gettext-0.21.1.tar.gz -> gettext-0.21.1.tar.gz...
-- Extracting source D:/vcpkg/vcpkg/downloads/gettext-0.21.1.tar.gz
-- Applying patch android.patch
-- Applying patch uwp.patch
-- Applying patch win-gethostname.patch
-- Applying patch 0003-Fix-win-unicode-paths.patch
-- Applying patch rel_path.patch
-- Using source at D:/vcpkg/vcpkg/buildtrees/gettext/src/gettext-0-de39ca5451.clean
-- Downloading https://github.com/lexxmark/winflexbison/releases/download/v2.5.25/win_flex_bison-2.5.25.zip -> win_flex_bison-2.5.25.zip...
-- Found external ninja('1.10.2').
-- Getting CMake variables for x86-windows
-- Using cached msys2-autoconf-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-automake-wrapper-20221207-1-any.pkg.tar.zst.
-- Using cached msys2-binutils-2.40-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libtool-2.4.7-3-x86_64.pkg.tar.zst.
-- Using cached msys2-make-4.4.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-pkgconf-1.9.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-which-2.21-4-x86_64.pkg.tar.zst.
-- Using cached msys2-bash-5.2.015-1-x86_64.pkg.tar.zst.
-- Using cached msys2-coreutils-8.32-5-x86_64.pkg.tar.zst.
-- Using cached msys2-file-5.44-5-x86_64.pkg.tar.zst.
-- Using cached msys2-gawk-5.2.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-grep-1~3.0-6-x86_64.pkg.tar.zst.
-- Using cached msys2-gzip-1.12-2-x86_64.pkg.tar.zst.
-- Using cached msys2-diffutils-3.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-sed-4.9-1-x86_64.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using cached msys2-autoconf2.71-2.71-1-any.pkg.tar.zst.
-- Using cached msys2-automake1.16-1.16.5-1-any.pkg.tar.zst.
-- Using cached msys2-libiconv-1.17-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libintl-0.21-2-x86_64.pkg.tar.zst.
-- Using cached msys2-zlib-1.2.13-1-x86_64.pkg.tar.zst.
-- Using cached msys2-tar-1.34-3-x86_64.pkg.tar.zst.
-- Using cached msys2-findutils-4.9.0-3-x86_64.pkg.tar.zst.
-- Using cached msys2-gmp-6.2.1-2-x86_64.pkg.tar.zst.
-- Using cached msys2-gcc-libs-11.3.0-3-x86_64.pkg.tar.zst.
-- Using cached msys2-libbz2-1.0.8-4-x86_64.pkg.tar.zst.
-- Using cached msys2-liblzma-5.4.1-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libzstd-1.5.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libreadline-8.2.001-3-x86_64.pkg.tar.zst.
-- Using cached msys2-mpfr-4.2.0-2-x86_64.pkg.tar.zst.
-- Using cached msys2-libpcre-8.45-3-x86_64.pkg.tar.zst.
-- Using cached msys2-m4-1.4.19-2-x86_64.pkg.tar.zst.
-- Using cached msys2-perl-5.36.0-1-x86_64.pkg.tar.zst.
-- Using cached msys2-ncurses-6.4-1-x86_64.pkg.tar.zst.
-- Using cached msys2-libcrypt-2.1-4-x86_64.pkg.tar.zst.
-- Using msys root at D:/vcpkg/vcpkg/downloads/tools/msys2/d7266db249278763
-- Using cached mingw-w64-i686-pkgconf-1~1.8.0-2-any.pkg.tar.zst.
-- Using cached msys2-msys2-runtime-3.4.6-1-x86_64.pkg.tar.zst.
-- Using msys root at D:/vcpkg/vcpkg/downloads/tools/msys2/6f3fa1a12ef85a6f
-- Configuring x86-windows-dbg
-- Configuring x86-windows-rel

**Comment by QiuYilin** [comment]
Try pressing enter.

**Reply by XiaoYou0426** [reply]
感谢回答，已经解决了，可能是卡住了，或者电脑不太好，重新加载几次就好了

**Reply by Suchnn** [reply]
> 感谢回答，已经解决了，可能是卡住了，或者电脑不太好，重新加载几次就好了

请问您是怎么解决的呀
